src/main/scala/uk/gov/hmrc/SbtArtifactory.scala

```diff
@@ -42,6 +42,7 @@ object SbtArtifactory extends sbt.AutoPlugin{
   private lazy val maybeArtifactoryPassword   = sys.env.get(artifactoryPasswordEnvKey)
 
   private lazy val maybeBintrayOrg            = sys.props.get("bintray.org")
+  private lazy val failOnBintrayError         = sys.props.get("bintray.fail_on_error").map(_ == "true").getOrElse(false)
```

Comment:
Since the unpublish doesn't fail on error, I had thought to keep the publish the same by default for symmetry. But happy to change.
I'll rename the param to `bintray.suppressError`